backend/routes/api/bookings.ts

```diff
@@ -51,4 +51,60 @@ router.get("/current", requireAuth, async (req, res) => {
 	return res.json({ Bookings: sequelized });
 });
 
+
+// delete booking by  bookingid
+
+const deleteBooking = (url: string, requireAuth: RequireAuth) => {
```

Comment:
nothing is importing this, it should be changed to 
```suggestion
router.delete("/:bookingId", requireAuth, async (req, res) => { ... })
```
where ... is the body of the handler